macros/utils/cross_db_utils/generate_elementary_profile_parameters.sql

```diff
@@ -0,0 +1,104 @@
+{% macro generate_elementary_profile_parameters(method=none, overwrite_values=none) %}
+  {#
+    Returns a list of parameters for elementary profile.
+    Each parameter consists of a list consisting of: 
```

Comment:
I tend to think that this should just be a list of dicts instead.

Reply:
sounds reasonable